src/main/java/net/dries007/tfc/config/ServerConfig.java

```diff
@@ -216,6 +234,7 @@
         ).define("enableForcedTFCGameRules", true);
         enableFireArrowSpreading = builder.apply("enableFireArrowSpreading").comment("Enable fire arrows and fireballs to spread fire and light blocks.").define("enableFireArrowSpreading", true);
         fireStarterChance = builder.apply("fireStarterChance").comment("Base probability for a firestarter to start a fire. May change based on circumstances").defineInRange("fireStarterChance", 0.5, 0, 1);
+        requireOffhandForRockKnapping = builder.apply("requireOffhandForRockKnapping").comment("If a rock is needed in the offhand in order to knap.").define("requireOffhandForRockKnapping", false);
```

Comment:
This is badly worded.
```suggestion
        requireOffhandForRockKnapping = builder.apply("requireOffhandForRockKnapping").comment(
            "If true, knapping with rocks will only work when one rock is held in each hand (main hand and off hand)",
            "If false, knapping with rocks will work either with main and off hand, or by holding at least two rocks in the main hand"
        ).define("requireOffhandForRockKnapping", false);
```